tests/generation/SettingsClassTest.py

```diff
@@ -9,6 +9,9 @@
     AllKindsOfSettingsDependentDecoratedBear)
 from tests.test_bears.AllKindsOfSettingsBaseBear import (
     AllKindsOfSettingsBaseBear)
+from tests.test_bears.AnotherLinterBear import AnotherLinterBear
+from tests.test_bears.SomeLinterBear import SomeLinterBear
```

Comment:
I was thinking if we can have more descriptive names for these bears, this would help in reusing them later.